client/src/components/organisms/Creator/Creator.jsx

```diff
@@ -8,15 +8,18 @@ import Typography from "./../../atoms/Typography/Typography"
 import {useAuth} from "./../../../context/auth-context"
 import {useVideo} from "./../../../context/video-context"
 import {searchFollowings} from "./../../../utils/context-utils/context-utils"
+import {followCreator, unfollowCreator} from "./../../../utils/api-calls/following" 
+import Spinner from "./../../atoms/Spinner/Spinner"
 import "./Creator.css"
 
 function Creator({kind}) {
     const { id } = useParams();
-    const [creatorVideos, setCreatorVideos] = useState("Loading...");
-    const [creatorDetails, setCreatorDetails] = useState("Loading...")
+    const [creatorVideos, setCreatorVideos] = useState(null);
+    const [creatorDetails, setCreatorDetails] = useState(null)
```

Comment:
We’re passing an empty string to `useState` to tell it that we want the value of the state variable we’re declaring to be empty by default.
An empty object implies data has been returned, whereas returning null clearly indicates that nothing has been returned.

Reply:
So shall I replace `null` with `{}`?